### PR TITLE
6.get comments by review

### PR DESCRIPTION
**Pull request by belindat98**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -134,8 +134,65 @@ describe("/api/reviews", () => {
...
+          expect(body.msg).toBe("Resource does not exist in reviews");
+        });
+    });
+    test("if valid review_id given but there are no comments for that id, returns 404 error with no comments found message", () => {
+      return request(app)
+        .get("/api/reviews/1/comments")
+        .expect(404)
+        .then(({ body }) => {
+          expect(body.msg).toBe("no comments found");
+        });
```
Comment (cjplatten) [inline __tests__/app.test.js:195]: Think about if this is a 404. You have sucessfully made a call to the database it just happens that there are no comments.

**Review by cjplatten — COMMENTED** [review]
Looking good so far - please see comments!

File: __tests__/app.test.js [hunk 2]
```diff
@@ -186,10 +186,10 @@ describe("/api/reviews/:review_id/comments", () => {
...
         });
     });
-    test("if valid review_id given but there are no comments for that id, returns 404 error with no comments found message", () => {
+    test("if valid review_id given but there are no comments for that id, returns 200 status with no comments found message", () => {
       return request(app)
         .get("/api/reviews/1/comments")
-        .expect(404)
+        .expect(200)
         .then(({ body }) => {
           expect(body.msg).toBe("no comments found");
```
Comment (HannahBurks) [inline __tests__/app.test.js]: In this case it would be more ideal to return and empty array

**Review by HannahBurks — CHANGES_REQUESTED** [review]
Hey Belinda, almost there, but in this case we would ideally like an empty array sent back for no comment - promise.reject makes it seem there is an error, however there is no error, there just aren't comments yet :)

**Review by dappernerddesigns — COMMENTED** [review]
Looks great to me, merge away

**Pull request merged**